docs/02.API-REFERENCE.md

```diff
@@ -2681,7 +2681,7 @@ jerry_value_is_object (const jerry_value_t value)
 Returns whether the given `jerry_value_t` is a promise value.
 
 *Notes*:
-- This API depends on a build option (`JERRY_BUILTIN_PROMISE`) and can be checked
+- This API depends on a build option (`JERRY_ESNEXT`) and can be checked
```

Comment:
This dependency change must be documented in this api ref. Please add a "Changed in version [[NEXT_RELEASE]]" entry (after the "new in version" entry).